namespaces/aushape.yml

```diff
@@ -0,0 +1,55 @@
+
+namespace:
+  name: aushape
+  type: group
+  description: >
+    Audit events converted with aushape
+    For more information please see
+    https://github.com/Scribery/aushape
+  fields:
+  - name: serial
+    type: long
+    description: >
+      Audit event serial number
+  - name: time
+    type: date
+    description: >
+      Audit event timestamp
+  - name: node
+    type: string
+    description: >
+      Name of the host where the audit event occurred
+  - name: error
+    type: string
+    description: >
+      The error aushape encountered while converting the event
+  - name: trimmed
+    type: string
+    description: >
+      An array of JSONPath expressions relative to the event object,
+      specifying objects/arrays with (some) contents removed as the result of
+      event size limiting. Empty string means event itself. Empty array means
+      trimming occurred at unspecified objects/arrays.
+  - name: text
+    type: string
+    index: analyzed
+    doc_values: false
+    description: >
+      An array log record strings representing the original audit event
```

Comment:
This field is an array of lines you can see in /var/log/audit.log, which belong to this particular event. My idea was to let people search them for specific words that can appear e.g. in file names. Perhaps I misunderstand "index: analyzed" and "not_analyzed" should be enough.